packages/miniflare/src/index.ts

```diff
@@ -1173,26 +1176,32 @@ export class Miniflare {
 
 			// Allow additional sockets to be opened directly to specific workers,
 			// bypassing Miniflare's entry worker.
-			const { unsafeDirectHost, unsafeDirectPort } = workerOpts.core;
-			if (unsafeDirectHost !== undefined || unsafeDirectPort !== undefined) {
-				const name = getDirectSocketName(i);
+			const previousDirectSockets =
+				previousWorkerOpts?.core.unsafeDirectSockets ?? [];
+			const directSockets = workerOpts.core.unsafeDirectSockets ?? [];
+			for (let j = 0; j < directSockets.length; j++) {
+				const previousDirectSocket = previousDirectSockets[j];
+				const directSocket = directSockets[j];
+				const entrypoint = directSocket.entrypoint ?? "default";
```

Comment:
Entirely not an issue — just a general question: I wonder if, instead of using `"default"` as a semantic value here, it would have been potentially a bit easier to allow `entrypoint` to be `undefined` and use that to mean the default export? I know `default` is a protected word, so no risk of a collision, but it just seems like you're needing to flip back and forth a bit here and in other files from that `"default"` value to the special case.

I also had a bit of an issue with this myself in the UI — React doesn't like undefined/null input values, so I went with an empty string, but same diff.